app/controllers/roles_controller.rb

```diff
@@ -24,4 +25,11 @@ def role_params
     params.require(:user).permit(:role)
   end
 
-end
+  def check_admin
+     if current_user.try(:admin?) 
+      flash[:success] = "Can assign roles"
```

Comment:
wtf is this @smarty1palak

Reply:
why are you displaying a flash message for a background check

Reply:
that was just to check if it's working
you can remove.

Reply:
Shouldn't you be removing such useless things you wrote for your convenience before sending a MR?